src/index.ts

```diff
@@ -15,3 +15,9 @@ export { InvocationContext } from './InvocationContext';
 export * as output from './output';
 export * as trigger from './trigger';
 export { Disposable } from './utils/Disposable';
+
+export enum SqlChangeOperation {

```

Comment:
Have we used enums before in this library? I couldn't find any other examples.

We could use a union type instead: `type SqlChangeOperation = 'insert' | 'update' | 'delete'`

Or, if we want to keep the enum, I think it's preferable to use strings for type safety and clarity:

```
export enum SqlChangeOperation {
    Insert = 'insert',
    Update = 'update',
    Delete = 'delete',
}

```

What do you think?

Reply:
See my other comment. I don't like enums (which is why you can't find other examples), but the sql extension is passing us 0, 1, and 2 for these values so I don't see an easy way around this

Reply:
I could do a union type, but it would have to be `type SqlChangeOperation = 0 | 1 | 2;` because again the sql extension is giving us these as numbers not strings. And I figured if we have to use numbers, then the enum is better